### PR TITLE
Update about.md

### PR DESCRIPTION
**Pull request by renbot-bio**
Added myself to the website! (Finally!)

### CODE REVIEW

File: markdown/about.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ The core team members who administer the nf-core project are listed below:
 * [@MaxUlysse](https://github.com/MaxUlysse): Maxime Garcia
 * [@olgabot](https://github.com/olgabot): Olga Botvinnik
 
+In addition, the following member is a part of the nf-core outreach team 
+* Renuka Kudva
```
Comment (MaxUlysse) [inline markdown/about.md:28]:
```suggestion
 * Renuka Kudva
```

File: markdown/about.md [hunk 2]
```diff
@@ -24,6 +24,9 @@ The core team members who administer the nf-core project are listed below:
 * [@MaxUlysse](https://github.com/MaxUlysse): Maxime Garcia
 * [@olgabot](https://github.com/olgabot): Olga Botvinnik
 
+In addition, the following member is a part of the nf-core outreach team
```
Comment (MaxUlysse) [inline markdown/about.md:27]:
```suggestion
In addition, the following member is a part of the nf-core outreach team
```

File: markdown/about.md [hunk 3]
```diff
@@ -24,6 +24,9 @@ The core team members who administer the nf-core project are listed below:
 * [@MaxUlysse](https://github.com/MaxUlysse): Maxime Garcia
 * [@olgabot](https://github.com/olgabot): Olga Botvinnik
 
+In addition, the following member is a part of the nf-core outreach team 
+ * Renuka Kudva
```
Comment (MaxUlysse) [inline markdown/about.md:28]:
```suggestion
* Renuka Kudva
```

File: markdown/about.md (hunk 1 above)
Comment (ewels) [reply markdown/about.md:28]: Why?

File: markdown/about.md [hunk 4]
```diff
@@ -24,6 +24,9 @@ The core team members who administer the nf-core project are listed below:
 * [@MaxUlysse](https://github.com/MaxUlysse): Maxime Garcia
 * [@olgabot](https://github.com/olgabot): Olga Botvinnik
 
+In addition, the following member is a part of the nf-core outreach team
+* Renuka Kudva
```
Comment (ewels) [inline markdown/about.md:28]:
```suggestion
In addition, the following member is a part of the nf-core outreach team

* [@renbot-bio](https://github.com/renbot-bio): Renuka Kudva
```

**Review by ewels — APPROVED** [review]
LGTM! 🎉

**Pull request merged**